packages/library-base/src/jvmMain/kotlin/io/realm/kotlin/internal/platform/StdOutLogger.kt

```diff
@@ -30,7 +34,20 @@ internal class StdOutLogger(
 
     override fun log(level: LogLevel, throwable: Throwable?, message: String?, vararg args: Any?) {
         val logMessage: String = prepareLogMessage(throwable, message, *args)
-        println("${level.name}: [$tag] $logMessage")
+        val timestamp: String = getTimestamp()
+        println("$timestamp ${level.name}: [$tag] $logMessage")
+    }
+
+    /**
+     * The `StdOutLogger` is only used on pure JVM, but is also included in our Android builds,
+     * which means that the use of `DateTimeFormatter` trigger warnings as it is only available
+     * from API 26+. Just suppress these warnings.
+     *
+     * We cannot use `DateFormatter` as it isn't thread-safe.
+     */
+    @Suppress("NewApi")
```

Comment:
Why does this trigger a warning on Android if it is only included on JVM target?

Reply:
Looks like it is just Android Studio getting a lint check wrong. Keeping it to avoid red lines inside Android Studio.